### DISCUSSION TITLE
bun update: get a changelog of updated packages

### DISCUSSION THREAD

**Discussion by mquandalle** [oven-sh/bun#14039] [Ideas]
A lot of packages authors publish a `changelog.md` file describing changes in each version. It would be great to display these change descriptions when updating packages with `bun update`.

**Comment by Jarred-Sumner** [comment]
would be nice

**Comment by mquandalle** [comment]
Dependabot does this in their update PRs. The implementation might be useful as reference:
  - [Changelog Finder](https://github.com/dependabot/dependabot-core/blob/main/common/lib/dependabot/metadata_finders/base/changelog_finder.rb)
  - [Changelog Pruner](https://github.com/dependabot/dependabot-core/blob/main/common/lib/dependabot/metadata_finders/base/changelog_pruner.rb)
